site/content/extend/plugins/webapp/_index.md

```diff
@@ -9,39 +9,44 @@ weight: 20
 
 Web app plugins are capable of overriding and extending React UI components of the Mattermost web and desktop apps.
 
-Looking for a quick start? [Check out our "Hello, world!" guide.](/extend/plugins/webapp/hello-world/)
+Looking for a quick start? [See our "Hello, world!" example](/extend/plugins/webapp/hello-world/).
 
-Want the web app plugin reference? [It can be found here.](/extend/plugins/webapp/reference/)
+Want the web app plugin reference doc? [Find it here](/extend/plugins/webapp/reference/).
 
 ## Features
 
 #### Override React Components
-By providing your own implementations of React components, replace the default Mattermost implementations to fit the needs of your plugin.
 
-For example, if you want to build a custom implemenation of the profile popover (sometimes called profile hovercard) that appears when a user's profile picture is clicked on, simply write your own implementation of the [ProfilePopover](/extend/plugins/webapp/reference/#profilepopover) component and include it in your plugin.
+You can provide your own implementations of React components to replace the default Mattermost components.
```

Comment:
This sounds a little weird to me, how about:

`Use your own implementations of web app React components to replace the default Mattermost implementations.`